src/Vault.sol

```diff
@@ -596,8 +755,8 @@ contract Vault is ERC4626, ERC20Permit, ILiquidationSource, Ownable {
    * @dev `_receiver` cannot be the zero address.
    */
   function _mint(address _receiver, uint256 _shares) internal virtual override {
-    _twabController.twabMint(_receiver, uint112(convertToAssets(_shares)));
-    super._mint(_receiver, _shares);
+    _assetSupplyBalance += convertToAssets(_shares);
+    _twabController.twabMint(_receiver, uint112(_shares));
```

Comment:
Vaults still need to be ERC20 compatible; this means it must emit the Transfer event as it would normally.

Only the *storage* is in the TwabController; the events still must be emitted here.

Reply:
Fixed in this commit: https://github.com/pooltogether/v5-vault/pull/6/commits/7831c1c1f0436ea59a445a9eae484b1e1356a380